ui/pages/confirmations/hooks/useConfirmationAlerts.ts

```diff
@@ -40,6 +41,7 @@ function useTransactionAlerts(): Alert[] {
   const signingOrSubmittingAlerts = useSigningOrSubmittingAlerts();
   ///: END:ONLY_INCLUDE_IF
   const queuedConfirmationsAlerts = useQueuedConfirmationsAlerts();
+  const selectedAccountAlerts = useSelectedAccountAlerts();
```

Comment:
Minor, should this go in `useConfirmationAlerts` below as it applies to signatures and transactions?

Reply:
I updated the PR.